src/pages/Admin/Details/ValidationSchema.ts

```diff
@@ -40,6 +44,37 @@ export interface RoomSchemaShape {
   image_url: string;
 }
 
+const venueNameLengthValidation = Yup.string()
+  .required("Venue name is required!")
+  .min(
+    VENUE_NAME_MIN_CHAR_COUNT,
+    ({ min }) => `Name must be at least ${min} characters`
+  )
+  .max(
+    VENUE_NAME_MAX_CHAR_COUNT,
+    ({ max }) => `Name must be less than ${max} characters`
+  );
```

Comment:
I'm almost certain we already have something like this.. from memory it was from one of @sunny-viktoryia / @malaikaaiyar 's PRs? Possibly still outstanding? Would be good to align with that.

Reply:
hmm can't recall anything I've been working on related to that. :thinking: 

But there were @mike-lvov's hot fixes recently where he increased the length of the name as far as I remember. 

Did you mean that?
https://github.com/sparkletown/sparkle/pull/1336

Reply:
I think I meant: https://github.com/sparkletown/sparkle/pull/1341